link updates (#1)
ottrproject/ottrproject.github.io: customize-docker.Rmd

We recommend using the `jhudsl/base_ottr:main` tagged image instead of the `latest` version. The `main` image is ready for use, while the `latest` may be under development. All GitHub Actions by default use the `jhudsl/base_ottr:main`.

> If you are new to Docker, you may find it helpful to read this [introduction to Docker](https://ottrproject.org/Adv_Reproducibility_in_Cancer_Informatics/launching-a-docker-image.html).

Reviewer: I don't think course links have changed

There's a library of ottr-ready docker images that you can see [here](https://github.com/jhudsl/ottr_docker). If you would like to contribute your own customized docker image that works with OTTR, please file a pull request!

1) Create a [Dockerhub account](https://hub.docker.com/signup) if you don't have one.

> 2) Create a new Docker image on your Dockerhub account. Follow these [instructions](https://ottrproject.org/Adv_Reproducibility_in_Cancer_Informatics/launching-a-docker-image.html) if you don't know how to do that.

Reviewer: ```suggestion
2) Create a new Docker image on your Dockerhub account. Follow these [instructions](https://jhudatascience.org/Adv_Reproducibility_in_Cancer_Informatics/launching-a-docker-image.html) if you don't know how to do that.
```

Then start a new branch so that you can submit a new pull request with your changes.

> Now, when you file a pull request, the Dockerfile build for your docker image will be tested automatically by the [GitHub actions](https://github.com/ottrproject/OTTR_Template/wiki/How-to-set-up-and-customize-GitHub-actions-robots) if `docker-test` is set to `yes`. The default is `docker-test: no`

Reviewer 2: https://www.ottrproject.org/customize-robots.html

> Now, when you file a pull request, the Dockerfile build for your docker image will be tested automatically by the [GitHub actions](https://github.com/ottrproject/OTTR_Template/wiki/How-to-set-up-and-customize-GitHub-actions-robots) if `docker-test` is set to `yes`. The default is `docker-test: no`

Reviewer: ```suggestion
Now, when you file a pull request, the Dockerfile build for your docker image will be tested automatically by the [GitHub actions](https://www.ottrproject.org/customize-robots.html) if `docker-test` is set to `yes`. The default is `docker-test: no`
```

### Modifying the Dockerfile for a new image

> [Read this chapter for instructions on how to modify Docker images](https://ottrproject.org/Adv_Reproducibility_in_Cancer_Informatics/modifying-a-docker-image.html)

Reviewer: ```suggestion
[Read this chapter for instructions on how to modify Docker images](https://jhudatascience.org/Adv_Reproducibility_in_Cancer_Informatics/modifying-a-docker-image.html)
```

### Testing a modified Docker image

> [Read this chapter for instructions on how to modify Docker images](https://ottrproject.org/Adv_Reproducibility_in_Cancer_Informatics/modifying-a-docker-image.html)

Reviewer: ```suggestion
[Read this chapter for instructions on how to modify Docker images](https://jhudatascience.org/Adv_Reproducibility_in_Cancer_Informatics/modifying-a-docker-image.html)
```
